tests/utests/basic/test_context.c

```diff
@@ -22,6 +22,36 @@
 #include "tree_schema_internal.h"
 #include "utests.h"
 
+#ifdef _WIN32
+static void slashes_to_backslashes(char *path)
+{
+    while ((path = strchr(path, '/'))) {
+        *path = '\\';
+    }
+}
+#endif
+
+static void assert_string_equal_backslashes(const char *expected_with_slashes, const char *tested_maybe_backslashed)
+{
+    char *tmp = strdup(expected_with_slashes);
+#ifdef _WIN32
+    slashes_to_backslashes(tmp);
+#endif
+    assert_string_equal(tmp, tested_maybe_backslashed);
+    free(tmp);
+}
+
+static LY_ERR ly_ctx_unset_searchdir_backslashes(struct ly_ctx *ctx, const char *value)
+{
+    char *tmp = strdup(value);
+#ifdef _WIN32
+    slashes_to_backslashes(tmp);
+#endif
+    LY_ERR ret = ly_ctx_unset_searchdir(ctx, tmp);
+    free(tmp);
+    return ret;
+}
+
```

Comment:
`MSVC: tests: use backslashes in expected normalized directory names` Again, perhaps rather skip this test altogether.

Reply:
Simplified now.